src/vmemcache.h

```diff
@@ -83,6 +91,7 @@ struct cache_entry {
 		struct repl_p_entry *p_entry;
 		size_t vsize;
 		VEC(, struct heap_entry) fragments;
+		stat_t updated;
```

Comment:
Since this consumes DRAM, there probably should be an option to #ifdef this out.

Reply:
So let's hold it